src/gui/qgsattributedialog.h

```diff
@@ -62,16 +62,18 @@ class GUI_EXPORT QgsAttributeDialog : public QDialog, public QgsMapLayerActionCo
      */
     void setHighlight( QgsHighlight *h );
 
-    QgsAttributeForm *attributeForm() { return mAttributeForm; }
+    /** Returns attribute form. */
```

Comment:
```suggestion
    /**
      * Returns the attribute form.
      */
```

(otherwise the python docstring is malformed)